app/utils/mdx.tsx

```diff
@@ -233,13 +233,41 @@ function FourOhFour() {
   )
 }
 
+function ServerError() {
```

Comment:
I'm not sure the `utils/mdx.tsx` util is the best place for this. Maybe `utils/misc.tsx`?

Reply:
Yeah, I just placed it next to the 404. How do you feel about moving them both to `components/errors.tsx`?

Reply:
Yeah, let's do 👍

Reply:
The 404 component in there was originally intended for blog posts and other mdx-based pages. But I guess it could be used for other things now too.